docs/api/guarding-policy-api.mdx

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 7
+title: Policy Guard API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The Policy Guard API allows you to manage policies at the organization level. Policy Guards help enforce centralized control over permissions and roles across specific projects included in the policy.
+For example, you can create  that enforces a specific set of permissions, as role admin can create documents, and role viewer can only read documents, across all projects included in the policy.
+
+Let's see how you can use the Policy Guard API for the example above.
+
+
+## Example API Usage
+
+
+### Create a New Policy Guard Scope
+```bash
+curl -X POST 'https://api.permit.io/v2/policy_guards/scopes' \
+-H 'authorization: Bearer API_SECRET_KEY' \
+--data-raw '{
+ key":   "policy_guard_acme",
+         "org_id": "c3b6f5d7-8b1e-4c6d-9e9f-8c9d6f8e0c8f",
+         "policy_guard_scope_details": [
+             {
+                 "env_id": "c3b6f5d7-8b1e-4c6d-9e9f-8c9d6f8e0c8f",
+                 "proj_id": "c3b6f5d7-8b1e-4c6d-9e9f-8c9d6f8e0c8f",
+             }]
+}'
+```
+### List all Policy Guard Scopes
```

Comment:
"Now, we'll want to list the policy guard scopes to see that it is created properly....